backend/src/services/chat-message/chat-message.service.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  ChatGetAllMessagesRequestDto,
+  ChatMessageCreateRequestDto,
+  ChatMessageGetAllItemResponseDto,
+  ChatMessageGetAllLastResponseDto,
+  ChatMessageGetAllResponseDto,
+  IdContainer,
+} from '~/common/types/types';
+import {
+  chatMessage as chatMessageRep,
+  menteesToMentors as menteesToMentorsRep,
+} from '~/data/repositories/repositories';
+import { createUuid, sanitizeHTML } from '~/helpers/helpers';
+
+type Constructor = {
+  chatMessageRepository: typeof chatMessageRep;
+  menteesToMentorsRepository: typeof menteesToMentorsRep;
+};
+
+class ChatMessage {
+  #chatMessageRepository: typeof chatMessageRep;
+
+  #menteesToMentorsRepository: typeof menteesToMentorsRep;
+
+  public constructor({
+    chatMessageRepository,
+    menteesToMentorsRepository,
+  }: Constructor) {
+    this.#chatMessageRepository = chatMessageRepository;
+    this.#menteesToMentorsRepository = menteesToMentorsRepository;
+  }
+
+  public async getAll({
+    chatId,
+  }: ChatGetAllMessagesRequestDto): Promise<ChatMessageGetAllResponseDto> {
+    const chatMessages = await this.#chatMessageRepository.getAll({
+      chatId,
+    });
+
+    return { items: chatMessages, chatId };
```

Comment:
What is the purpose of returning `chatId` back, if you're receiving it as argument?

Reply:
For the state, because then I would have to check an array with messages and here I get a separate property